.github/ISSUE_TEMPLATE/feature.yml

```diff
@@ -0,0 +1,18 @@
+name: Request a new feature
+description: New demands? Tell us what Pelorus could do to help you
+labels: ["kind/feature", "feature"]
```

Comment:
"kind/feature" is enough. We should add also "needs-triage". Drop "feature" as it's not something that prow is using.